CHANGELOG.md

```diff
@@ -28,6 +28,52 @@ Refactoring:
 
 **Full Changelog**: https://github.com/...
 
+# 0.59.0
+
+**Important news and breaking changes**
+
+This release brings a bunch of breaking changes:
+* Setting `roundingPeriod` for discovery jobs is deprecated, a warning will be logged at startup.
+* Setting `delay` at the metric level is deprecated, a warning will be logged at startup.
```

Comment:
```suggestion
* Setting `delay` at the metric level is deprecated, a warning will be logged at startup. This `delay` configuration has existed for a long time but was never utilized. Instead of utilizing it the decision was made to deprecate it as a way of simplifying the configuration.
```